working-groups/wg-pitr.md

```diff
@@ -0,0 +1,30 @@
+# PITR Working Group
+
+PITR is an ecosystem tool for TiDB Binlog. By preprocessing the incremental backup file of TiDB, PITR merged the changes of the same line of data to produce a new, lighter incremental backup file, which greatly reduced the Time of incremental backup Recovery and realized fast-pitr (Fast Point in Time Recovery).
+
+## Stakeholder SIGs
+
+To be discussed.
+
+## Meetings
+
+To be discussed.
+
+## Organizers
+
+* WangXiangUSTC (**[@wangxiangustc](https://github.com/WangXiangUSTC)**), PingCAP
+* HuaiyuXu (**[@huaiyuxu](https://github.com/XuHuaiyu)**), PingCAP
+* AilinKid (**[@AilinKid](https://github.com/AilinKid)**), PingCAP
+* tsthght(**[@tsthght](https://github.com/tsthght)**), MeiTuan
+
+## Contact
+- Slack: [#wg-pitr](https://tidbcommunity.slack.com/archives/CRH5594F8)
+- [Open Community Issues/PRs](https://github.com/pingcap/community/issues/126)
+
+## Published Documents
+
+https://github.com/lvleiice/Better-PITR
+
+### RFC/Proposal
```

Comment:
I think it's included in the `Published Documents`, maybe this section can be removed